scripts/start/prepare.R

```diff
@@ -334,201 +347,31 @@ prepare <- function() {
-                             list(c("q47_emiTarget_netCO2.M", "!!q47_emiTarget_netCO2.M")),
-                             list(c("q47_emiTarget_netCO2_noBunkers.M", "!!q47_emiTarget_netCO2_noBunkers.M")),
-                             list(c("q47_emiTarget_netCO2_noLULUCF_noBunkers.M", "!!q47_emiTarget_netCO2_noLULUCF_noBunkers.M")),
-                             list(c("q47_emiTarget_netGHG.M", "!!q47_emiTarget_netGHG.M")),
-                             list(c("q47_emiTarget_netGHG_noBunkers.M", "!!q47_emiTarget_netGHG_noBunkers.M")),
-                             list(c("q47_emiTarget_netGHG_noLULUCF_noBunkers.M", "!!q47_emiTarget_netGHG_noLULUCF_noBunkers.M")),
-                             list(c("q47_emiTarget_netGHG_LULUCFGrassi_noBunkers.M", "!!q47_emiTarget_netGHG_LULUCFGrassi_noBunkers.M")),
-
-                             list(c("q47_emiTarget_grossEnCO2.M", "!!q47_emiTarget_grossEnCO2.M")),
-                             list(c("q47_emiTarget_mkt_netCO2.M", "!!q47_emiTarget_mkt_netCO2.M")),
-                             list(c("q47_emiTarget_mkt_netCO2_noBunkers.M", "!!q47_emiTarget_mkt_netCO2_noBunkers.M")),
-                             list(c("q47_emiTarget_mkt_netCO2_noLULUCF_noBunkers.M", "!!q47_emiTarget_mkt_netCO2_noLULUCF_noBunkers.M")),
-                             list(c("q47_emiTarget_mkt_netGHG.M", "!!q47_emiTarget_mkt_netGHG.M")),
-                             list(c("q47_emiTarget_mkt_netGHG_noBunkers.M", "!!q47_emiTarget_mkt_netGHG_noBunkers.M")),
-                             list(c("q47_emiTarget_mkt_netGHG_noLULUCF_noBunkers.M", "!!q47_emiTarget_mkt_netGHG_noLULUCF_noBunkers.M")),
-                             list(c("q47_emiTarget_mkt_netGHG_LULUCFGrassi_noBunkers.M", "!!q47_emiTarget_mkt_netGHG_LULUCFGrassi_noBunkers.M")),
-                             list(c("qm_balFeAfterTax.M", "!!qm_balFeAfterTax.M")),
-                             list(c("q47_implicitQttyTargetTax.M", "!!q47_implicitQttyTargetTax.M")),
-                             list(c("q47_implEnergyBoundTax.M", "!!q47_implEnergyBoundTax.M")),
-                             list(c("vm_taxrevimplEnergyBoundTax.M", "!!vm_taxrevimplEnergyBoundTax.M"))
-                             )
-
-    fixings_manipulateThis <- c(fixings_manipulateThis,
-                            list(c("v47_emiTarget.FX", "!!v47_emiTarget.FX")),
-                            list(c("v47_emiTargetMkt.FX", "!!v47_emiTargetMkt.FX")),
-                            list(c("vm_taxrevimplEnergyBoundTax.FX", "!!vm_taxrevimplEnergyBoundTax.FX")))
+    file.copy("levs.gms", "fixings.gms", overwrite = TRUE)
+    # Replace fixings with level values
+    manipulateFile("fixings.gms", list(c(".L ", ".FX ")), fixed = TRUE)
 
     # Include fixings (levels) and marginals in full.gms at predefined position
     # in core/loop.gms.
-    full_manipulateThis <- c(full_manipulateThis,
+    full_manipulateThis <- c(
                              list(c("cb20150605readinpositionforlevelfile",
```

Comment:
Where does this actually go? I couldn't find any spot where this part will be inserted.

Reply:
I deleted it. It was removed here: https://github.com/remindmodel/remind/commit/a64823c01adf87d72abf789d16e6ac91861a604b